bank/src/main/java/com/clouway/bricky/adapter/http/HttpServletModule.java

```diff
@@ -0,0 +1,15 @@
+package com.clouway.bricky.adapter.http;
+
+import com.clouway.bricky.adapter.http.filter.SecurityFilter;
+import com.google.inject.servlet.ServletModule;
+
+/**
+ * @author Marian Zlatev <mzlatev91@gmail.com>
+ */
+public class HttpServletModule extends ServletModule {
+
+  @Override
+  protected void configureServlets() {
+    filter("*").through(SecurityFilter.class);
```

Comment:
This way I guarantee that all my rest service's are protected. When I create a new one I won't have to come and add it to the filter. The only accessible pages are Login and Register for now. If my logic changes maybe I would have to exclude some pages not to be filtered.